libs/s3/s3_test_lib.py

```diff
@@ -426,7 +426,7 @@ def delete_multiple_objects(
         """
         try:
             LOGGER.info("deleting multiple objects")
-            if prepared_obj_list:
+            if prepared_obj_list is not None:
```

Comment:
You could have just said as 
if not prepared_obj_list:

Reply:
That doesn't work here as [] is a valid value for prepared_obj_list